perl_lib/EPrints/Extras.pm

```diff
@@ -304,7 +304,7 @@ sub render_possible_doi
 
 	$value = "" unless defined $value;
 	if( $value =~ m!^
-			(?:http://(?:dx\.)?doi\.org/)?  # add this again later anyway
+			(?:http(?:s)://(?:dx\.)?doi\.org/)?  # add this again later anyway
```

Comment:
Yes... I think you're right.  And now to work out how to update a pull request...